optuna/trial/_trial.py

```diff
@@ -378,25 +380,59 @@ def objective(trial):
                 Upper endpoint of the range of suggested values. ``high`` is included in the range.
             step:
                 A step of discretization.
+
+                .. note::
+                    Note that :math:`\\mathsf{high}` is modified if the range is not divisible by
+                    :math:`\\mathsf{step}`. Please check the warning messages to find the changed
+                    values.
+
+                .. note::
+                    The method returns one of the values in the sequence
+                    :math:`\\mathsf{low}, \\mathsf{low} + \\mathsf{step}, \\mathsf{low} + 2 *
+                    \\mathsf{step}, \\dots, \\mathsf{low} + k * \\mathsf{step} \\le
+                    \\mathsf{high}`, where :math:`k` denotes an integer.
+
+                .. note::
+                    The ``step != 1`` and ``log`` arguments cannot be used at the same time.
+                    To set the ``step`` argument :math:`\\mathsf{step} \\le 2`, set the
```

Comment:
I guess the inequality sign is reversed.

![image](https://user-images.githubusercontent.com/3255979/84346964-3cdd7780-abec-11ea-8cfd-2de74ad07d1e.png)

```suggestion
                    To set the ``step`` argument :math:`\\mathsf{step} \\ge 2`, set the
```